cove_ocds/templates/cove_ocds/explore_release.html

```diff
@@ -341,6 +341,11 @@ <h4 class="panel-title">
         </h4>
       </div>
       <div class="panel-body">
+        {% if releases|length > 25 %}
+        <p>
+          {%blocktrans%}Only showing the first 25 releases. If you want to see more, look at the spreadsheet in "Convert" above.{%endblocktrans%}
```

Comment:
@Bjwebb don't we normally put spaces in `{% template commands %}`?

As for copy, I think:

"Showing the first 25 releases. To explore all your data in a tabular format, convert it to a spreadsheet using the "Convert" section, above".  

@jpmckinney , how does that sound to you?

Reply:
@robredpath Sounds good!

Reply:
> @Bjwebb don't we normally put spaces in {% template commands %}?

Normally. Altough I copied this from further down the file. I'll update this along with the text.